### PR TITLE
Add ood

### PR DESCRIPTION
**Pull request by daviddanan**
Several modifications regarding the airfoil usecase:

* Add architecture for the neural network identical to the one used in the NeurIps paper
* Add standard scaler where the mean/std are computed iteratively and where several variable are not normalized  
* Specify variable not to normalize
* Copy updated parameter value passed to predict in model
* Define ood dataset and filter for training data

### CODE REVIEW

File: configurations/airfoil/simulators/torch_fc.ini
Comment (Mleyliabadi) [inline configurations/airfoil/simulators/torch_fc.ini:1]: On pouvait ajouter une nouvelle section pour les configurations de l'architecture souhaitée ou bien créer un nouveau fichier config pour ne pas écraser les configs déjà existants.

**Review by Mleyliabadi — CHANGES_REQUESTED** [review]
If you can integrate the config changes into a new file (e.g., `torch_fc_airfrans`) or a new section dedicated to the AirfRANS use case `[CONFIG_AIRFRANS]`.

File: configurations/airfoil/simulators/torch_fc.ini
Comment (Mleyliabadi) [reply configurations/airfoil/simulators/torch_fc.ini:1]: Non c'est bon en fait, je n'avais pas fait attention que c'était dans l'acheminement propre au cas d'usage AirFoil: `[configurations/airfoil/simulators/torch_fc.ini]`
